tests/e2e/specFiles/Transactions/receive.spec.ts

```diff
@@ -115,8 +116,8 @@ describe('Transactions', () => {
     describe('Standard P2SH', () => {
       beforeEach(async () => {
         await steps.createWallet({
-          type: 'Standard HD P2SH',
-          name: 'Standard P2SH',
+          type: WalletType.S_HD_P2SH,
```

Comment:
Shouldnt we use WalletType.S_P2SH here? Currently all types are walletType.S_HD_P2SH

Reply:
Yes, fixed